scripts/deployments/validate_simcore_stack_yml.bash

```diff
@@ -52,6 +52,8 @@ do
     echo "Assuming targetfile in ${SETTINGS_BINARY_PATH}/${TARGETFILE}"
     echo "Checking ${SETTINGS_BINARY_PATH}/${TARGETFILE}"
     if docker compose --file ${COMPOSE_FILE} run --rm "${service}" test -f "${SETTINGS_BINARY_PATH}"/"${TARGETFILE}" > /dev/null 2>&1; then
+        # Pull image from registry, just in case
+        docker compose --file ${COMPOSE_FILE} pull
```

Comment:
I think you might want to add ${service} as well.
so you don't pull everything everytime